Hmwk#4_CSV/readcsv.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import csv
+import os
+import sys
+import re
+from typing import List, Union
+
+#--filename    -f    Path to a file containing a list of users to scrape.
+#--columnname   -f    name of column to read from csv file.
+
+def getArgs(argv = None): 
+    parser = argparse.ArgumentParser(description='Add filepath to file to read from.') #add arguments
+    parser.add_argument('-f','--filepath', type=str,help='path to file needed to read')
+    parser.add_argument('-c','--columnname', type=str,help='column from csv file needed to read')
+
+    return parser.parse_args(argv) #return arguments passed from cmd
+
+
+
+def checkFile(args):
+    #check for file
+    path_csv  = args.filepath #path to directory c:\smth or /smth
+    path_folder = os.path.split(path_csv)[0] #get path to folder only
+
+    filename = os.path.basename(path_csv).split('.')[0] #name without extension to check if such actually exists
+    extension = '.' + os.path.basename(path_csv).split('.')[1] #file extension in format .smth
+    
+    columnname =  re.sub(r'[^a-zA-Z0-9]',r' ', args.columnname)  #column name to retrieve data from, 
+                                                                 #remove '_' undersocre to resolve column names with 'space' input for cmd issue
+    
+    csv_to_read = None #initialze value for result file to read
+
+    result: List[Union[bytes, str]] = []            #store path + file, if exists in passed directory
+
+    for dirroot, dirname, filenames in os.walk(path_folder): #run through directory and retrieve filenames
+        for file in filenames:                      #check each filename
+            if extension == os.path.splitext(file)[1]:   #if file extension is what we're looking for
+                result.append(os.path.join(dirroot, file))  #add filename to direcory path to work with
+        
+        if len(result) < 1:     #if no files found - nothing to process
+            print('"{}" folder is empty'.format(dirroot))
+
+        for item in result:
+            if os.path.split(item)[1] == os.path.basename(path_csv):
+                csv_to_read = item    
+
+    if csv_to_read == None:
```

Comment:
Compare values to None using the pattern ` if cond is None:` according to https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html